docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -1,420 +1,104 @@
-----
-
-==== `symbol`
-
-Returns the ticker symbol of the token.
-
-Parameters: None.
-
-Returns:
-
-[,cairo]
-----
-symbol: felt
-----
-
-==== `decimals`
-
-Returns the number of decimals the token uses - e.g.
-`8` means to divide the token amount by `100000000` to get its user representation.
-
-Parameters: None.
-
-Returns:
-
-[,cairo]
-----
-decimals: felt
-----
-
-==== `totalSupply`
-
-Returns the amount of tokens in existence.
-
-Parameters: None.
-
-Returns:
-
-[,cairo]
-----
-totalSupply: Uint256
-----
-
-==== `balanceOf`
-
-Returns the amount of tokens owned by `account`.
-
-Parameters:
-
-[,cairo]
-----
-account: felt
-----
-
-Returns:
-
-[,cairo]
-----
-balance: Uint256
-----
-
-==== `allowance`
-
-Returns the remaining number of tokens that `spender` will be allowed to spend on behalf of `owner` through `transferFrom`.
-This is zero by default.
-
-This value changes when `approve` or `transferFrom` are called.
-
-Parameters:
-
-[,cairo]
-----
-owner: felt
-spender: felt
-----
-
-Returns:
-
-[,cairo]
-----
-remaining: Uint256
-----
-
-==== `transfer`
-
-Moves `amount` tokens from the caller's account to `recipient`.
-It returns `1` representing a bool if it succeeds.
-
-Emits a <<transfer_event,Transfer>> event.
-
-Parameters:
-
-[,cairo]
-----
-recipient: felt
-amount: Uint256
-----
-
-Returns:
-
-[,cairo]
-----
-success: felt
-----
-
-==== `transferFrom`
-
-Moves `amount` tokens from `sender` to `recipient` using the allowance mechanism.
-`amount` is then deducted from the caller's allowance.
-It returns `1` representing a bool if it succeeds.
-
-Emits a <<transfer_event,Transfer>> event.
-
-Parameters:
-
-[,cairo]
-----
-sender: felt
-recipient: felt
-amount: Uint256
-----
-
-Returns:
-
-[,cairo]
-----
-success: felt
-----
-
-==== `approve`
-
-Sets `amount` as the allowance of `spender` over the caller's tokens.
-It returns `1` representing a bool if it succeeds.
-
-Emits an <<approval_event,Approval>> event.
-
-Parameters:
-
-[,cairo]
-----
-spender: felt
-amount: Uint256
-----
-
-Returns:
-
-[,cairo]
-----
-success: felt
-----
-
-=== Events
-
-[,cairo]
-----
-func Transfer(from_: felt, to: felt, value: Uint256) {
-}
-
-func Approval(owner: felt, spender: felt, value: Uint256) {
+#[external(v0)]
+fn mint(self: @ContractState, to: ContractAddress, amount: u256) {
+    let unsafe_state = ERC20::unsafe_new_contract_state();
+    
+    // Ensures the mint function is protected!
+    assert_only_owner(); 
+    ERC20::InternalImpl::_mint(@unsafe_state, to, amount);
 }
 ----
 
-==== `Transfer (event)`
+WARNING: This mechanism is unsafe because storage members could clash among used contracts if not reviewed carefully. Extensibility will be revisited after {components} are introduced.
 
-Emitted when `value` tokens are moved from one account (`from_`) to another (`to`).
+== Presets
 
-Note that `value` may be zero.
+The ERC20 contract itself includes a basic implementation preset and can be used as-is for quick prototyping and testing.
```

Comment:
That's fair. The idea was to have the intro paragraph with the expectation of more presets to come (I think it would less redundant then). I'll remove